CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 ### Changed
-
+-   #1355 : remove BasicParser arguments.
```

Comment:
BasicParser is code so it should be in code quotes:
```suggestion
-   #1355 : Remove unused `BasicParser` arguments.
```

Reply:
The start of sentences should be capitalised. I think it is also useful to mention that these arguments are unused so this is not a big change